### PR TITLE
clean: Force sax parser implementation

### PR DESCRIPTION
**Pull request by rtfpessoa**
This is needed to support AOT

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -1,54 +1,32 @@
-import Dependencies._
-
 name := "coverage-parser"
 
-version := "1.1.1-SNAPSHOT"
-
-scalaVersion := "2.11.11"
+scalaVersion := "2.11.12"
```
Comment (andreaTP) [inline build.sbt]: this line can be removed

File: build.sbt [hunk 2]
```diff
@@ -1,54 +1,32 @@
...
+pgpPassphrase := Option(System.getenv("SONATYPE_GPG_PASSPHRASE")).map(_.toCharArray)
 
-organizationHomepage := Some(new URL("https://www.codacy.com"))
+resolvers ~= { _.filterNot(_.name.toLowerCase.contains("codacy")) }
 
-publishMavenStyle := true
+publicMvnPublish
 
-publishArtifact in Test := false
+organization := "com.codacy"
```
Comment (andreaTP) [inline build.sbt]: this comes from the sbt plugin

File: build.sbt [hunk 3]
```diff
@@ -1,54 +1,32 @@
...
 
-pomIncludeRepository := { _ => false}
+organizationName := "Codacy"
```
Comment (andreaTP) [inline build.sbt]: this comes from the sbt plugin

File: build.sbt [hunk 4]
```diff
@@ -1,54 +1,32 @@
...
 
-publishTo := {
-  val nexus = "https://oss.sonatype.org/"
-  if (version.value.trim.endsWith("SNAPSHOT"))
-    Some("snapshots" at nexus + "content/repositories/snapshots")
-  else
-    Some("releases" at nexus + "service/local/staging/deploy/maven2")
-}
+organizationHomepage := Some(new URL("https://www.codacy.com"))
```
Comment (andreaTP) [inline build.sbt]: if this doesn't come from the sbt plugin it should :-)

File: project/Dependencies.scala [hunk 5]
```diff
@@ -6,10 +6,12 @@ object Dependencies {
     val scalaApi = "com.codacy" %% "codacy-api-scala" % "3.1.869"
     val pluginsApi = "com.codacy" %% "codacy-plugins-api" % "3.0.276"
   }
+
   def scalaXml(scalaVersion: String): Seq[ModuleID] = scalaVersion match {
     case "2.10.7" => Seq.empty
     case _ => Seq("org.scala-lang.modules" %% "scala-xml" % "1.1.1")
   }
+  
   val scalaTest = "org.scalatest" %% "scalatest" % "3.0.4" % "test"
```
Comment (andreaTP) [inline project/Dependencies.scala]: scalaTest comes from the sbt plugin

File: project/build.properties [hunk 6]
```diff
@@ -1 +1 @@
-sbt.version=0.13.16
+sbt.version=0.13.17
```
Comment (andreaTP) [inline project/build.properties]: can't we bump to 1.2.X?

File: project/plugins.sbt [hunk 7]
```diff
@@ -1,21 +1,7 @@
-import sbt._
+resolvers += "Era7 maven releases" at "https://s3-eu-west-1.amazonaws.com/releases.era7.com"
```
Comment (andreaTP) [inline project/plugins.sbt]: this is not needed since you are filtering codacy resolvers in build.sbt

Comment (rtfpessoa) [reply project/plugins.sbt]: it fails

**Pull request merged**